app/views/parks/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1> Parks Index test </h1>
```

Comment:
You might as well give this it's "real" name, rather than a name just for the test :)